src/directives/map.js

```diff
@@ -18,12 +19,14 @@ goog.require('ol.Map');
  * [../examples/simple.html](../examples/simple.html)
  *
  * @htmlAttribute {ol.Map} ngeo-map The map.
+ * @param {ngeo.SyncDataSourcesMap} ngeoSyncDataSourcesMap Ngeo sync
+ *     data sources map service.
  * @return {angular.Directive} Directive Definition Object.
  * @ngInject
  * @ngdoc directive
  * @ngname ngeoMap
  */
-ngeo.mapDirective = function() {
+ngeo.mapDirective = function(ngeoSyncDataSourcesMap) {
```

Comment:
The `ngeo.FeatureOverlayMgr` seems have its map set at the "application" level, i.e. in the controllers or in the examples.

I think we always want the `ngeoSyncDataSourcesMap` to do its magic, i.e. synchronize the `inRange` property, otherwise you have to make sure you do not forget about it.  If we're okay with that, then fine.  I personnally think it would be a dangerous move.  If you forget about it, in an example (for example) it would become very hard for a third person to understand why it doesn't behave like it should (i.e. a data source that's supposed to be out of range would always be).  I think we should leave it there for that reason.

What do you think?